code/modules/clothing/clothing.dm

```diff
@@ -18,9 +18,6 @@
 
 /obj/item/clothing/Initialize(mapload, ...)
```

Comment:
Remove this completely.

Reply:
Sure I can do that. Trying to work out how to resolve the whole clothing in sorter aspect at the moment. I may just make clothing its own rule for sorters because of this whole edge-case affecting different things in different areas.